### PR TITLE
Update SignalR CORS for 5.0

### PR DESCRIPTION
**Pull request by BrennanConroy**
Fixes https://github.com/dotnet/AspNetCore.Docs/issues/15636

And adds a 5.0 setting that was commented out.

### CODE REVIEW

File: aspnetcore/signalr/configuration.md [hunk 1]
```diff
@@ -12,7 +12,393 @@ uid: signalr/configuration
...
+| `WebSocketConfiguration` | `null` | A delegate that can be used to configure additional WebSocket options. Receives an instance of [ClientWebSocketOptions](/dotnet/api/system.net.websockets.clientwebsocketoptions) that can be used to configure the options. |
+
+# [JavaScript](#tab/javascript)
+
+| JavaScript Option | Default Value | Description |
+| ----------------- | ------------- | ----------- |
+| `accessTokenFactory` | `null` | A function returning a string that is provided as a Bearer authentication token in HTTP requests. |
+| `skipNegotiation` | `false` | Set this to `true` to skip the negotiation step. **Only supported when the WebSockets transport is the only enabled transport**. This setting can't be enabled when using the Azure SignalR Service. |
+| `withCredentials` | `true` | Specifies whether credentials will be sent with the CORS request. Azure App Service uses Cookies for sticky sessions and needs this option enabled to work correctly. For more info on CORS with SignalR see <xref:signalr/security#cross-origin-resource-sharing> |
```
Comment (BrennanConroy) [inline aspnetcore/signalr/configuration.md:346]: This is the new addition for this file

Comment (scottaddie) [inline aspnetcore/signalr/configuration.md:346]:
```suggestion
| `withCredentials` | `true` | Specifies whether credentials will be sent with the CORS request. Azure App Service uses cookies for sticky sessions and needs this option enabled to work correctly. For more info on CORS with SignalR, see <xref:signalr/security#cross-origin-resource-sharing>. |
```

**Comment by BrennanConroy** [comment]
> Having to copy the entire section before modifying it does not make reviewing easy.

I put a comment on the part I changed in that file https://github.com/dotnet/AspNetCore.Docs/pull/18000#discussion_r414834963
Although since it's outdated it wont show when you're viewing the file

**Comment by scottaddie** [comment]
@halter73 I agree. This problem will disappear once we implement folder-level versioning.

**Pull request merged**